spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/AbstractBinder.java

```diff
@@ -254,13 +260,11 @@ private Object deserializePayload(byte[] bytes, MimeType contentType) {
 					this.payloadTypeCache.put(className, targetType);
 				}
 				return this.codec.decode(bytes, targetType);
-			}
-			catch (ClassNotFoundException e) {
-				throw new SerializationFailedException("unable to deserialize [" + className + "]. Class not found.",
-						e); // NOSONAR
-			}
-			catch (IOException e) {
-				throw new SerializationFailedException("unable to deserialize [" + className + "]", e);
+			}// catch all exceptions that could occur during de-serialization
+			catch (Exception e) {
```

Comment:
Hope combining all the exceptions here makes sense as the exception messages appears redundant.